.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: build
+
+on:
+  push:
+
+jobs:
+  test:
+      runs-on: macos-latest
```

Comment:
Do we need macos for a specific reason? ...can we use ubuntu as well?
Asking because macos seems to be the problematic environment (like payment stuff recently) so I wonder if there are specific constraints. On Android, I used macos because it was the one that worked better with the emulator at that time.

Reply:
Linux is actually not supported by the tracker... There is an open issue for it and it seems that the only problem is making the HTTP client compatible with Linux, but currenty the tracker only supports Windows and macOS.

Reply:
From a payment point of view, macos is not a problem in an Open Source repo :-)

Reply:
Payment was probably a bad example but the easiest to remember. :) My general point is that Apple is always quite restrictive and I remember I had issues years ago running Micro on macos because of Docker problems (probably something solved now).